storage/innobase/log/log0recv.cc

```diff
@@ -2233,7 +2233,7 @@ static byte *recv_parse_or_apply_log_rec_body(
 
       ut_ad(!page || page_type != FIL_PAGE_TYPE_ALLOCATED || page_no == 0);
 
-#ifndef UNIV_HOTBACKUP
+#if !defined(UNIV_HOTBACKUP) && !defined(XTRABACKUP)
```

Comment:
I see that we also parse the encryption markers for ALTER TABLESPACE (see space->encryption_op_in_progress) but looks like it doesn't cause harm as of now?

Can you please add testcase for all these scenarios?
1. stop checkpoint_thread, page_cleaner ALTER TABLESPACE ENCRYPT='Y' take backup
2. stop checkpoint_thread, page_cleaner, ALTER TABLESPAC ENCRYPT='N' take backup
3. stop checkpoint_thread, page_cleaner, ALTER TABLESPACE ENCRYPT='Y', then N and then Y, take backup (this bug)